pbm/config.go

```diff
@@ -269,6 +269,28 @@ func (p *PBM) SetConfigVar(key, val string) error {
 	return errors.Wrap(err, "write to db")
 }
 
+func (p *PBM) DeleteConfigVar(key string) error {
+	if !ValidateConfigKey(key) {
+		return errors.New("invalid config key")
+	}
+
+	_, err := p.GetConfig()
+	if err != nil {
+		if errors.Cause(err) == mongo.ErrNoDocuments {
```

Comment:
please use `errors.Is()` function. `errors.Cause()` returns a direct underlying error in a chain. `mongo.ErrNoDocuments` could be not direct but wrapped by another somewhere in `GetConfig()`

Reply:
Actually I copied this piece of code from `GetConfigVar` above.
But I'm ready to change both of them for consistency.

Reply:
you are welcome to change both :)

Reply:
Done.